src/Snppts/Snippets/FaceBookApp.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using Snppts.Authors;
+using Snppts.Extensions;
+using Snppts.Infrastructure;
+
+namespace Snppts.Snippets
+{
+    public class FaceBookApp : IAmASnippet
+    {
+        public string Slug => "FacebookApp";
+        public string Title => "Facebook App";
+        public string GithubRepoName => "leomarisreyes/FaceBookUISample";
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+
+        public string Description => "It's the replica of Facebook App.";
+
+        public IAmAnAuthor AuthorInfo => new LeomarisReyes();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://askxammy.com/wp-content/uploads/2019/08/FB_post.png"),
```

Comment:
Please supply these images as individual images per screen. Putting 3 screens in the same image messes up the layout of the site ☺️